From 1ac121f9f3ff1ac59ed1c11d398daaee2346c57c Mon Sep 17 00:00:00 2001
From: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
Date: Tue, 10 Mar 2026 15:55:50 +0800
Subject: [PATCH 15/15] plat: imx: fix definition conflict of IMX_SIP_CPU_OFF

The value 0xC200000D has been used by IMX_SIP_BBSM,
change IMX_SIP_CPU_OFF to use 0xC2000012.

Signed-off-by: Hou Zhiqiang <Zhiqiang.Hou@nxp.com>
Upstream-Status: Pending
---
 plat/imx/common/include/imx_sip_svc.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plat/imx/common/include/imx_sip_svc.h b/plat/imx/common/include/imx_sip_svc.h
index 6406154f5..a9d8bb129 100644
--- a/plat/imx/common/include/imx_sip_svc.h
+++ b/plat/imx/common/include/imx_sip_svc.h
@@ -69,8 +69,6 @@
 
 #define IMX_SIP_MISC_SET_TEMP		0xC200000C
 
-#define IMX_SIP_CPU_OFF			0xC200000D
-
 #if defined(PLAT_imx93)
 #define IMX_SIP_BBSM			0xC200000D
 #define IMX_SIP_BBSM_CLEAR_INTERRUPT	0x01
@@ -98,6 +96,8 @@ int imx_configure_memory_for_vpu(void *handle, u_register_t x1);
 int imx_get_partition_number(void *handle);
 #endif
 
+#define IMX_SIP_CPU_OFF			0xC2000012
+
 #if defined(PLAT_imx8mq)
 int imx_soc_info_handler(uint32_t smc_fid, u_register_t x1,
 			 u_register_t x2, u_register_t x3);
-- 
2.43.0

